pkg/backend/backend.go

```diff
@@ -0,0 +1,70 @@
+package backend
```

Comment:
`pkg/backend` seems like an odd path for this rather than something like `backend/`. can you explain this to me tomorrow?